internal/util-interface/src/main/java/xsbti/TextEdit.java

```diff
@@ -0,0 +1,25 @@
+/*
+ * sbt
+ * Copyright 2011 - 2018, Lightbend, Inc.
+ * Copyright 2008 - 2010, Mark Harrah
+ * Licensed under Apache License 2.0 (see LICENSE)
+ */
+
+package xsbti;
+
+/**
+ * A representation of the `TextEdit` found in the LSP protocol.
+ *
+ * <p>NOTE: That instead of a `Range` we use the internal [[xsbti.Position]].
+ *
+ * @see <a
+ *     href="https://microsoft.github.io/language-server-protocol/specifications/lsp/3.17/specification/#textEdit">`TextEdit`</a>
+ */
+public interface TextEdit {
+
+  /** The position this edit will be applied to. */
+  Position position();
```

Comment:
Just to make sure, the position has a start and an end? So it's basically a range?